### PR TITLE
stat/distuv: Add quantile and survival functions to Gamma, ChiSquared…

### PR DESCRIPTION
**Pull request by btracey**
…, and F

### CODE REVIEW

File: distuv/chisquared.go [hunk 1]
```diff
@@ -72,11 +72,27 @@ func (c ChiSquared) Rand() float64 {
...
 func (c ChiSquared) StdDev() float64 {
 	return math.Sqrt(c.Variance())
 }
 
+// Survival returns the survival function (complementary CDF) at x.
+func (c ChiSquared) Survival(x float64) float64 {
+	if x < 0 {
+		return 1
+	}
+	return mathext.GammaIncComp(c.K/2, x/2)
```
Comment (sbinet) [inline distuv/chisquared.go]: personal pet peeve: replace div-by-2 with mul-by-0.5 :)

File: distuv/chisquared.go [hunk 2]
```diff
@@ -72,11 +72,27 @@ func (c ChiSquared) Rand() float64 {
 	return Gamma{c.K / 2, 0.5, c.Src}.Rand()
 }
 
+// Quantile returns the inverse of the cumulative distribution function.
+func (c ChiSquared) Quantile(p float64) float64 {
+	if p < 0 || p > 1 {
+		panic(badPercentile)
+	}
+	return mathext.GammaIncInv(c.K/2, p) * 2
```
Comment (sbinet) [inline distuv/chisquared.go]: personal pet peeve: replace div-by-2 with mul-by-0.5 :)

File: distuv/chisquared.go [hunk 3]
```diff
@@ -72,11 +72,27 @@ func (c ChiSquared) Rand() float64 {
 	return Gamma{c.K / 2, 0.5, c.Src}.Rand()
```
Comment (sbinet) [inline distuv/chisquared.go]: personal pet peeve: replace div-by-2 with mul-by-0.5 :)

**Comment by btracey** [comment]
I fixed the ones that are related to this commit. PTAL

**Comment by btracey** [comment]
Ping?

**Pull request merged**